src/main/scala/cromwell/engine/backend/package.scala

```diff
@@ -3,6 +3,11 @@ package cromwell.engine
 import scala.concurrent.{ExecutionContext, Future}
 
 package object backend {
+  // Ordered by shards, and then ordered by attempts
+  type AttemptedCallLogs = Seq[Seq[CallLogs]]
```

Comment:
we should probably get in the habit of ensuring our `Seq` references are immutable, you can import scala.collections.imutable.Seq to do that